daemon/src/model/cfd.rs

```diff
@@ -2272,6 +2272,31 @@ mod tests {
         assert_eq!(funding_fee.rate, funding_rate);
     }
 
+    use proptest::prelude::*;
+    proptest! {
+        #[test]
+        fn rollover_funding_fee_collected_incrementally_should_not_be_smaller_than_collected_once_per_settlement_interval(amount_sat in 1_000u64..1_000_000u64) {
+            let funding_rate = FundingRate::new(dec!(0.01)).unwrap();
+
+            let funding_fee_for_whole_interval =
+                calculate_funding_fee(Amount::from_sat(amount_sat), funding_rate, SETTLEMENT_INTERVAL.whole_hours()).unwrap();
+            let funding_fee_for_one_hour =
+                calculate_funding_fee(Amount::from_sat(amount_sat), funding_rate, 1).unwrap();
+            let fee_account = FeeAccount::new(Position::Long, Role::Taker);
+
+            let fee_account_whole_interval = fee_account.add_funding_fee(funding_fee_for_whole_interval);
+            let fee_account_one_hour = fee_account.add_funding_fee(funding_fee_for_one_hour);
+
+            let total_balance_when_collected_hourly = fee_account_one_hour.balance().checked_mul(SETTLEMENT_INTERVAL.whole_hours()).unwrap();
+            let total_balance_when_collected_for_whole_interval = fee_account_whole_interval.balance();
+
+            prop_assert!(
+                total_balance_when_collected_hourly >= total_balance_when_collected_for_whole_interval,
```

Comment:
Better to subtract values from each other and assert that the diff is smaller than some small number.

Reply:
I derived some value by tried and error (if we'd like to make it tighter, we can always adjust the implementation). 30 sat seems good for now.